ddp-workspace/projects/ddp-sdk/src/lib/models/activity/MailAddressBlock.ts

```diff
@@ -31,7 +31,5 @@ export class MailAddressBlock extends ActivityBlock {
         return true;
     }
 
-    public shouldScrollToFirstInvalidQuestion(): boolean {
```

Comment:
Because the MailAddressBlock class extends from the ActivityBlock class and this method is already defined elsewhere, this one was eliminated. I do not believe it is necessary to create an override method that only returns false. We can reconsider it if it was designed for another purpose, though, as it is now causing problems.

Reply:
Historically, we block "scroll to first invalid question" feature for ContentBlock, InstitutionBlock and for **MailAddressBlock**. 

@MocanaAtBroad do you remember whether it has done in purpose ? 
I mean for the Mailing address.

I suppose we would like to have that feature for Mailing Address as well (according to the Juan's fix).